### PR TITLE
patch roles for user-api to pull logs

### PR DESCRIPTION
**Pull request by harshad16**
## Related Issues and Dependencies

Fixes: #459 

## This introduces a breaking change

- [x] No

## Test or Stage Environment

- [x] Pull Requests added content to STAGE environment

## This Pull Request implements

allow user-api to reach pod logs in backend.

## Description

patch roles for user-api to pull logs
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

File: user-api/overlays/stage/role-binding.yaml [hunk 1]
```diff
@@ -15,9 +15,23 @@ subjects:
...
+  - kind: ServiceAccount
+    name: user-api
+    namespace: thoth-frontend-stage
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
 metadata:
   name: user-api-argo
-  namespace: thoth-frontend-stage
+  namespace: thoth-middletier-stage
```
Comment (pacospace) [inline user-api/overlays/stage/role-binding.yaml:34]: middletier?

Comment (harshad16) [reply user-api/overlays/stage/role-binding.yaml:34]: this was a missed in the manifest earlier
user-api doesn't create any workflow in frontend, this is for middletier , as package-extract is scheduled there.

Comment (pacospace) [reply user-api/overlays/stage/role-binding.yaml:34]: from now on user-api does not schedule anything, it won't need to talk to other namespaces, investigator will do its actions. I guess we can remove some service accounts and role bindings now

Comment (harshad16) [reply user-api/overlays/stage/role-binding.yaml:34]: ah i see, then we can remove these parts. 
other content is need for retrieving the adviser pod logs.
about the role for schedule we can remove them in separate pr.

Comment (pacospace) [reply user-api/overlays/stage/role-binding.yaml:34]: Sounds good to me, thanks Harshad!!

**Review by pacospace — APPROVED** [review]
lgtm

**Pull request merged**